### PR TITLE
Updating event.html

### PR DESCRIPTION
**Pull request by cassmarcussen**
Updating event.html to be more specific to the website.

### CODE REVIEW

File: js/event.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 function submit() {
-    alert("fuck you");
```
Comment (JamesYang007) [inline js/event.js:2]: why did you delete this?

**Review by JamesYang007 — APPROVED** [review]
eh

**Review by mab2400 — APPROVED** [review]
**<3**

File: js/event.js (hunk 1 above)
Comment (cassmarcussen) [reply js/event.js:2]: apologies, thought we should give the user a clearer message
